File: android/testData/folding/plurals.java
package p1.p2;

import android.app.Activity;

public class Plurals extends Activity {
    public void test(int quantity) <fold text='{...}' expand='true'>{
        String s = <fold text='"Hello!"' expand='false'>getResources().getQuantityString(R.plurals.plural1, quantity)</fold>;
        String t = <fold text='Quantity One' expand='false'>getResources().getQuantityString(R.plurals.plural2, quantity)</fold>;
    }</fold>
}

File: android-kotlin/idea-android/testData/android/folding/plurals.kt
package com.myapp

import android.content.Context

fun Context.testPlurals(quantity: Int) <fold text='{...}' expand='true'>{
    val plural1 = <fold text='"Reference One"' expand='false'>resources.getQuantityString(R.plurals.first, quantity)</fold>
    val plural2 = <fold text='"Quantity One"' expand='false'>this.getResources().getQuantityString(R.plurals.second, quantity)</fold>
    val plural3 = <fold text='"{3} song found from {25}."' expand='false'>resources.getQuantityString(R.plurals.numberOfSongsAvailable, quantity, 3, 25)</fold>
}</fold>

